frontend/src/context/analysisResultStateSlice.ts

```diff
@@ -317,6 +443,40 @@ const createAPIRequestParams = (
   return apiRequest;
 };
 
+function mergeTableRows(tableRows: TableRow[]) {
+  const mergedObject: Record<string, any> = {};
+
+  tableRows.forEach(obj => {
```

Comment:
we could probably simplify the function by using something like:

```
mergeWith({}, ...Object.values(objectThatNeedsToCollapse), (objValue, srcValue) =>
typeof objValue === "number" ? objValue + srcValue : undefined
)
```

Reply:
@ericboucher I changed a little bit the code there by using `array.reduce`.